File: libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/milestone/MilestoneEventListener.java
package com.mapbox.services.android.navigation.v5.milestone;

import com.mapbox.services.android.navigation.v5.routeprogress.RouteProgress;

public interface MilestoneEventListener {

  void onMilestoneEvent(RouteProgress routeProgress, String instruction, Milestone milestone);

}

File: libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/milestone/MilestoneEventListener.kt
package com.mapbox.services.android.navigation.v5.milestone

import com.mapbox.services.android.navigation.v5.routeprogress.RouteProgress

interface MilestoneEventListener {

    fun onMilestoneEvent(
        routeProgress: RouteProgress,
        instruction: String,
        milestone: Milestone
    )
}
